.github/workflows/test_iree.yml

```diff
@@ -77,15 +77,13 @@ jobs:
         run: |
           source ${VENV_DIR}/bin/activate
           python3 iree_tests/download_remote_files.py --root-dir iree_tests/pytorch/models
-          python3 iree_tests/download_remote_files.py --root-dir iree_tests/sharktank
 
       - name: "Running real weight model tests"
         if: "matrix.models-config-file != '' && !cancelled()"
         run: |
           source ${VENV_DIR}/bin/activate
           pytest \
             iree_tests/pytorch/models \
```

Comment:
I can start porting these to https://github.com/iree-org/iree-test-suites as well. Just opt-125M and resnet50 in https://github.com/nod-ai/SHARK-TestSuite/tree/main/iree_tests/pytorch/models right now since no one helped port from e2eshark x_x, we wouldn't lose much by dropping that coverage without a replacement.